src/collective/ifttt/tests/robot/test_content_trigger.robot

```diff
@@ -0,0 +1,135 @@
+
+# ============================================================================
+# CONTENT TRIGGER ROBOT TESTS
+# ============================================================================
+#
+# Run this robot test stand-alone:
+#
+#  $ bin/test -s collective.ifttt -t test_content_trigger.robot --all
+#
+# Run this robot test with robot server (which is faster):
+#
+# 1) Start robot server:
+#
+# $ bin/robot-server --reload-path src collective.ifttt.testing.COLLECTIVE_IFTTT_ACCEPTANCE_TESTING
+#
+# 2) Run robot tests:
+#
+# $ bin/robot src/collective/ifttt/tests/robot/test_content_trigger.robot
+#
+#   for debug mode
+# $ bin/robot-debug src/collective/ifttt/tests/robot/test_content_trigger.robot
+
+# See the http://docs.plone.org for further details (search for robot
+# framework).
+#
+# ============================================================================
+
+*** Settings ***
+
+Resource  plone/app/robotframework/selenium.robot
+Resource  plone/app/robotframework/keywords.robot
+Resource  Selenium2Screenshots/keywords.robot
+
+Library  Remote  ${PLONE_URL}/RobotRemote
+Library  Selenium2Screenshots
+
+Test Setup  Test Setup
+Test Teardown  Close all browsers
+
+*** Variables ***
+
+${SELENIUM2LIBRARY_RUN_ON_FAILURE}  Capture page screenshot
+${SCREENSHOTS}  false
+
+
+*** Test Cases ***
+
+Scenario: As a site administrator I can see Content Trigger actions form
+    Given I'm logged in as a Site Administrator
+    And I fill secret key
+    And I goto home page
+   When I trigger the 'Add IFTTT Content Trigger' action menu item
+   Then check 'This will add new IFTTT Trigger' on pagecontent
+
+Scenario: As a site administrator I can configure Content Trigger action
+    Given I'm logged in as a Site Administrator
+    And I fill secret key
+    And I goto home page
+   And I trigger the 'Add IFTTT Content Trigger' action menu item
+   When I fill Content Trigger form
+   Then I see ' Successfully applied the IFTTT event test_event to Plone site' on success
+
```

Comment:
@Shriyanshagro Please,

* [ ] fix indentation in these scenarios
* [ ] why there is leading space in ' Successfully ...'

Reply:
* [ ]  the current branch contains unnecessary empty file  `src/collective/ifttt/tests/test_content_trigger.py`